frontend/src/store/modules/visitors/actions.js

```diff
@@ -40,6 +46,34 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [CHANGE_FETCHED_LINE_CHART_STATE]: (context, value) => {
+
+        if (value) {
+            context.commit(SET_LINE_CHART_FETCHING);
+        } else {
+            context.commit(RESET_LINE_CHART_FETCHING);
+        }
+    },
+    [FETCH_LINE_CHART_DATA]: (context, data) => {
+        if (!data.value) {
+            return;
+        }
+        context.commit(SET_LINE_CHART_FETCHING);
+
+        getTimeByPeriod(context.state.selectedPeriod)
```

Comment:
again, `getTimeByPeriod` doesn't return Promise, it returns object. So it won't have `then` method, so this action won't work